BUG: fix bug related to windowing in forward operator and fbp.

aringh opened the pull request:
Fix for #30

aringh committed:
BUG: fix bug related to windowing in forward operator and fbp.

adler-j commented on adutils.py:
```diff
@@ -134,16 +134,22 @@ def get_ray_trafo(reco_space, use_subset=False, use_rebin=False,
-                                  for Ai in A])
+        # This only works for A a BroadcastOperator, and as long as the
+        # Tam-Danielsson window is a vector/matrix and not an operator
+        if isinstance(A[0], odl.OperatorLeftVectorMult):
```
I would probabily prefer something like

```
if isinstance(A[0], odl.OperatorLeftVectorMult):
    ops = [Ai.operator for Ai in A]
    vecs = [Ai.vector for Ai in A]
else
    ops = A
    vecs = [odl.tomo.tam_danielson_window(Ai, smoothing_width=0.1,
                                          n_half_rot=3)
            for Ai in A]
            
fbp = odl.ReductionOperator(*[(odl.tomo.fbp_op(
        opi, padding=True, filter_type='Hamming',  # Hann
        frequency_scaling=0.8) * wi) for opi, wi in zip(ops, vecs)])
```

since maintainance becomes easier

adler-j reviewed (approved):
One stylistic thing, else fine

aringh committed:
MAINT: changes in the get_fbp method.

Changes in the get_fbp method, to check if there is a
Tam-Danielsson window with the forward operator.

aringh commented:
Changed according to comments. Should be good to go :-)

davlars commented:
Looks good. Nice work!
